lib/mail_room/mailbox.rb

```diff
@@ -92,6 +95,10 @@ def ssl_options
       replace_verify_mode(ssl)
     end
 
+    def context
+      "[#{self.email}-#{self.name}]:"
```

Comment:
What about using a hash here? The advantage of structured logging would be easier to leverage fi we'd merge this context into the structured log.

```suggestion
      { email: self.email, name: self.name }
```